### PR TITLE
LRDOCS-5336 screenshots

### PR DESCRIPTION
**Pull request by nealrobbins**

### CODE REVIEW

**Comment by nealrobbins** [comment]
https://issues.liferay.com/browse/LRDOCS-5336

File: discover/portal/articles/190-managing-users/01-users-and-organizations/02-advanced-user-management.markdown [hunk 1]
```diff
@@ -78,7 +78,7 @@ default icons:
 
 1.  Navigate to *Control Panel* &rarr; *Configuration* &rarr; *System Settings*. 
 
-2.  Find the Users category. Click *User Images*.
```
Comment (rbohl) [inline discover/portal/articles/190-managing-users/01-users-and-organizations/02-advanced-user-management.markdown]: FYI @nealrobbins Users is the category. Platform is the section.

File: discover/portal/articles/190-managing-users/01-users-and-organizations/02-advanced-user-management.markdown [hunk 2]
```diff
@@ -147,14 +147,14 @@ names conflicted with site IDs. In @product-ver@, this property defaults to
 
     users.screen.name.allow.numeric=true
 
-This means you're free to set a user screen name to *97854*, or whatever other
+This means you're free to set a user screen name to *24601*, or whatever other
```
Comment (rbohl) [inline discover/portal/articles/190-managing-users/01-users-and-organizations/02-advanced-user-management.markdown]: @nealrobbins Out of curiosity, why did you change the number?

**Comment by rbohl** [comment]
Sent for Publication: https://github.com/sez11a/liferay-docs/pull/3391

File: discover/portal/articles/190-managing-users/01-users-and-organizations/02-advanced-user-management.markdown (hunk 2 above)
Comment (nealrobbins) [reply discover/portal/articles/190-managing-users/01-users-and-organizations/02-advanced-user-management.markdown]: No, sorry, it's not important. I was trying to get find-and-replace commands to stick.

Comment (rbohl) [reply discover/portal/articles/190-managing-users/01-users-and-organizations/02-advanced-user-management.markdown]: Haha, nice. My working theories were

1. 97854 exactly matches your salary and you didn't want it in the docs.
2. 97854 was your ID number at the penitentiary and you didn't want to see it in the docs.
3. 24601 was your ID number at the penitentiary and you wanted it in the docs.
